server/datastore/mysql/operating_systems_test.go

```diff
@@ -101,6 +102,35 @@ func TestUpdateHostOperatingSystem(t *testing.T) {
 	require.Equal(t, true, isSameOS(t, testOS, *storedOS))
 }
 
+func TestUniqueOS(t *testing.T) {
+	ctx := context.Background()
+	ds := CreateMySQLDS(t)
+
+	testHostIDs := make([]uint, 20)
```

Comment:
As the size of the test slice approaches 100, we start to see "Error 1040: Too many connections". Do we handle those errors generally when ingesting agent data?

Reply:
I don't think we handle those errors in any way, no.